example/src/eyedrop/getCanvasBlockColors.js

```diff
@@ -0,0 +1,28 @@
+// returns an array of rgb objects [{r:10, g:10, b:10}]
+const getCanvasBlockColors = (canvas, sx, sy, sw, sh) => {
```

Comment:
Great job on this one! Would absolutely love if you could cover it with a unit test or three. :smiley:

Reply:
I don't know how to write unit test. I tested on my project with Chrome's network inspect, set a very large radius and record the processing time. The original took 5+ seconds, this one took around 300ms.

Reply:
I see...
Oh well, I could write those unit tests, but might need your assistance in fully grasping it.
I will make sure to ping you here when I do. :smile: